console_conf/ui/views/chooser.py

```diff
@@ -144,56 +151,67 @@ def __init__(self, controller, systems):
             Pile([heading_table, systems_table]),
         ]
 
-        buttons = [
-            reset_btn("ABORT", on_press=self.abort),
-        ]
+        buttons = []
+        if controller.model.current is not None:
+            # back to options of current system
+            buttons.append(back_btn("BACK", on_press=self.back))
 
-        super().__init__(screen(
-            rows=rows,
-            buttons=button_pile(buttons),
-            focus_buttons=False,
-            excerpt=self.excerpt))
+        super().__init__(controller.model.current,
+                         screen(
+                             rows=rows,
+                             buttons=button_pile(buttons),
+                             focus_buttons=False,
+                             excerpt=self.excerpt))
 
     def _system_action(self, sender, action, system):
         self.controller.select(system, action)
 
-    def abort(self, result):
-        self.controller.cancel()
+    def back(self, result):
+        self.controller.back()
 
 
-class ChooserConfirmView(BaseView):
-    title = "Ubuntu Core"
-    excerpt = ("Summary of the selected action.")
+class ChooserConfirmView(ChooserBaseView):
+
+    canned_summary = {
+        "run": "Continue running the system without any changes.",
```

Comment:
@degville would appreciate your input on the messages here. This is presented to the user as a summary of the selected action.

The `default_summary` is a fallback messaage we use for an action that we don't know about.

Reply:
As I know you want to merge this, I've had a quick glance over the text and it looks good.

The only thing I'd probably tweak is the default summary to something like 'Make sure you understand the consequences of performing this action.`, or maybe add a little more help (or a link) on what those consequences might be.